week05/clause_test.go

```diff
@@ -0,0 +1,82 @@
+package demo
+
+import (
+	"testing"
+)
+
+func emptyClauses(t testing.TB) *Clauses {
+	t.Helper()
+	return &Clauses{}
+}
```

Comment:
There is no need for this helper. It is shorter in the code to just write `Clauses{}`